examples/particles/js/editor/controls.js

```diff
@@ -472,7 +469,6 @@
                 code.push("emitter.name = '" + emitter.name + "';");
                 code.push("emitter.z = " + emitter.z + ";");
```

Comment:
what should be it be updated to ? here above you also refer to `emiter.z` (line 46/47)